hpx/runtime/actions/basic_action.hpp

```diff
@@ -861,6 +861,7 @@ namespace hpx { namespace serialization
 #define HPX_REGISTER_ACTION_1(action)                                         \
     HPX_REGISTER_ACTION_2(action, action)                                     \
 /**/
+#if defined(HPX_MSVC) || defined(HPX_MINGW)
```

Comment:
Why does this get rid of any warnings? It looks to me like the if and else branches are exactly the same...

Reply:
Good catch ... looks like I didn't commit the latest version...